lab-nicholasc/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
you should have winston faker and net installed locally using `npm i -S faker winston net`